src/client/components/pages/parts/call-to-action.js

```diff
@@ -41,35 +43,35 @@ function CallToAction() {
 				<ButtonGroup id="searchpage-button-group">
 					<Button
 						className="padding-bottom-1 padding-sides-2 padding-top-1"
-						href="/author/create"
+						href={props.query ? `/author/create?name=${props.query}` : '/author/create'}
```

Comment:
A suggestion to make this a bit more readable:
Since the name will be the same for each route, we can create a variable once at the start of this component function:
```const nameQueryParameter = props.query ? `?name=${props.query}` : "";```

And then use it like so for each route below:
```href={`/author/create${nameQueryParameter}`}```